modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/installing-with-agent-based-installer.adoc
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-ocp-agent-ibm-z_{context}"]
+= Manually adding {ibm-z-title} agents
+
+After creating the PXE assets you can add {ibm-z-name} agents. Depending on your {ibm-z-name} environment you can choose from the following:
+
+* Adding {ibm-z-name} agents with z/VM
+* Adding {ibm-z-name} agents with {op-system-base} KVM
+
+.Procedure
+
+** Adding {ibm-z-name} agents with z/VM
+
+. Create a parameter file for the z/VM guest.
++
+.Example parameter file
++
+[source,terminal]
+----
+rd.neednet=1 \
+console=ttysclp0 \
+coreos.live.rootfs_url=<rootfs_url> \ <1>
+ip=172.18.78.2::172.18.78.1:255.255.255.0:::none nameserver=172.18.78.1 \ <2>
+zfcp.allow_lun_scan=0 \ <3>
+rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
+rd.dasd=0.0.4411 \ <4>
+rd.zfcp=0.0.8001,0x50050763040051e3,0x4000406300000000 \ <5>
```

Comment:
Remove either rd.dasd or rd.zfcp. Keep only one here and mention other one in callout.

Reply:
I prefer to keep as is to stay in sync with what we have for [SNO](https://docs.openshift.com/container-platform/4.14/installing/installing_sno/install-sno-installing-sno.html#installing-sno-on-ibm-z_install-sno-installing-sno-with-the-assisted-installer)